gammapy/datasets/map.py

```diff
@@ -107,9 +107,11 @@ def create_map_dataset_geoms(
 
 
 class MapDataset(Dataset):
-    """
-    Bundle together binned counts, background, IRFs, models and compute a likelihood.
-    It uses the Cash statistics by default.
+    """Main map dataset for likelihood fitting.
+    It bundles together binned counts, background, IRFs into `~gammapy.maps.WcsNDMap`.
+    A safe mask and a fit mask can be added to exclude bins during the analysis.
```

Comment:
I just realise that when we never describe them correctly!
I propose that we add a subsection in https://docs.gammapy.org/dev/user-guide/datasets/index.html in an other PR...

Reply:
This is part of the dataset tutorial: https://docs.gammapy.org/1.0.1/tutorials/api/datasets.html#using-masks

But I agree it would be good to add a section in the user-guide as well.

Reply:
Yes, of course. I was wondering if it was wise to make a reference to a tutorial into the docstring of a function... I have no strong argument against it. I would slightly prefer to mention it in the user guide. 
Opinions?